internals/testing/polyfills.js

```diff
@@ -1,14 +0,0 @@
-if (typeof Promise === 'undefined') {
-  // Rejection tracking prevents a common issue where React gets into an
-  // inconsistent state due to an error, but it gets swallowed by a Promise,
-  // and the user has no idea what causes React's erratic future behavior.
-  require('promise/lib/rejection-tracking').enable();
-  window.Promise = require('promise/lib/es6-extensions.js');
-}
-
-// fetch() polyfill for making API calls.
-require('whatwg-fetch');
-
-// Object.assign() is commonly used with React.
-// It will use the native implementation if it's present and isn't buggy.
-Object.assign = require('object-assign');
```

Comment:
By removing this file, the polyfill for Promise and Object.assign is also removed. I'm not sure about the current state of support for those methods in major browsers, but it should be considered adding them to the webpack entry point if they are still required.

Reply:
This file was only used for testing from what I can tell. I think Object.assign should be pretty well supported now, but I'll take a look. Babel might already transform it like it does with some other features.

Reply:
Ah, I didn't notice that it was just used for testing. The project officially supports Node 8+, so no need for the polyfills anymore :1st_place_medal: